scripts/build-tools.js

```diff
@@ -14,7 +14,7 @@ const buildTools = async () => {
       resolve(__dirname, '../config', 'tools-automated.json'),
       JSON.stringify(automatedTools, null, '  ')
     );
-    await combineTools(automatedTools, manualTools);
+    await combineTools(automatedTools, manualTools, resolve(__dirname, '../../config', 'tools.json'), resolve(__dirname, '../../config', 'all-tags.json'));
```

Comment:
Can we do it in a more better way, instead of passing filePath here? Try to refactor tools scripts instead of manually passing filePaths for tags file.